packages/donation-form/src/form-elements/contact-form/contact-form.ts

```diff
@@ -411,17 +411,14 @@ export class ContactForm extends LitElement {
         }
 
         contact-form #donation-contact-form-countryCodeAlpha2 {
-          width: ${iconFieldWidth};
+          width: calc(100%);
```

Comment:
Q: Can we just do `100%` without the `calc`?

Reply:
maybe?